mite_http/__init__.py

```diff
@@ -81,7 +81,7 @@ def response_callback(r):
                 first_byte_time=r.starttransfer_time,
                 total_time=r.total_time,
                 primary_ip=r.primary_ip,
-                method=r.request.method,
+                method=r.request.method.decode("utf-8"),
```

Comment:
i avoided doing this for efficiency reasons.  i don't have benchmarks to show how much it matters, but the theory was that avoiding extra work was a good thing.  if `mite cat` is the only use case that cares, then i'd prefer to fix it there.  (i seem to remember that some prometheus metrics got uglier too, as in `method="b'GET'"` instead of just `method="GET"`...idk if that was ever fixed but it would be presumably trivial to do so in the prometheus metrics code.  the point is not to do it on the hot path.)

Reply:
I did some benchmarking and it's pretty negligible, but still impacting. I think I agree, if it only breaks `mite cat` it's not worth it and this PR will allow `mite cat` to output byte encoding values anyway.